flow-typed/webpackDefinePlugins.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Makes flow aware of global variables defined by the webpack define plugin.
```

Comment:
this is a global attached to the window, but it's not fed by webpack. comment may confuse?

Reply:
Oh true! Yes, oops. I'll rename the module to `flow-typed/globals.js`.